pkg/controller/component/rsm_convertor.go

```diff
@@ -112,13 +113,47 @@ func (c *rsmUpdateStrategyConvertor) convert(args ...any) (any, error) {
 	}
 	if getMemberUpdateStrategy(synthesizedComp) != nil {
 		// appsv1.OnDeleteStatefulSetStrategyType is the default value if member update strategy is set.
```

Comment:
remove this comment

Reply:
fixed in: [851aed9](https://github.com/apecloud/kubeblocks/pull/6776/commits/851aed915d2d9441ac101643776390a6417f5ad1)